packages-web/signature/development.md

```diff
@@ -0,0 +1,72 @@
+# Development and contribution
```

Comment:
Remove development.md should be generic whole repo?

Reply:
Fixed